package.json

```
@@ -15,7 +15,9 @@
     "generate-svg-mono-icons": "svgr --config-file .svgrrc.mono-icons.yml -d ./src/assets/svg/icons ./assets/svgs/mono-icons && prettier --write ./src/assets/svg/icons",
     "generate-svg-color": "svgr --config-file .svgrrc.color.yml -d ./src/assets/svg/ ./assets/svgs/color && prettier --write ./src/assets/svg/",
     "generate-svgs": "yarn generate-svg-mono-icons && yarn generate-svg-color",
-    "update-device-profiles": "bundle exec fastlane ios update_devices"
+    "update-device-profiles": "bundle exec fastlane ios update_devices",
+    "clean": "rm -rf node_modules/ yarn.lock **/node_modules/ **/yarn.lock **/package-lock.json  yarn-error.log",
```

```suggestion
    "clean": "rm -rf node_modules/ **/node_modules/ yarn-error.log",
```

Greit med scriptet her, men vi bør aldri slette/recreate `.lock`-filer med mindre vi absolutt må.